### PR TITLE
Improvements to UI

### PR DESCRIPTION
**Pull request by joehawary**
Includes fix for CRU-216, CRU-201, CRU-190

### CODE REVIEW

File: src/components/layout/footer.tsx [hunk 1]
```diff
@@ -61,6 +61,12 @@ const Footer: FC = () => {
       </Box>
       <Box py={2}>
         <HStack spacing={4}>
+          <ChakraLink
```
Comment (nuurcodes) [inline src/components/layout/footer.tsx:64]: Might be good to add `rel='noopener noreferrer'` for security

Comment (joehawary) [reply src/components/layout/footer.tsx:64]: Good suggestion! Added 👍

File: src/styles/global-styles.ts [hunk 2]
```diff
@@ -9,4 +9,25 @@ export const GlobalStyles = css`
...
+      circle,
+      rgba(69, 73, 105, 1) 0%,
+      rgba(27, 25, 39, 1) 80%
+    );
+    background: radial-gradient(
+      circle,
+      rgba(69, 73, 105, 1) 0%,
+      rgba(27, 25, 39, 1) 80%
+    );
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#454969",endColorstr="#1b1927",GradientType=1);
```
Comment (nuurcodes) [inline src/styles/global-styles.ts:30]: Is this filter needed?

File: src/styles/button.ts [hunk 3]
```diff
@@ -5,6 +5,19 @@ export const Button = {
...
+          bg: 'purple.100',
+        },
+      },
+      _active: {
+        bg: 'purple.100',
+      },
+    },
+    cyanbutton: {
       bg: 'cyan.300',
       color: 'purple.800',
```
Comment (nuurcodes) [inline src/styles/button.ts:22]: maybe `gray.800` is better here

File: src/store/rewardPrograms.ts [hunk 4]
```diff
@@ -44,22 +44,22 @@ const rewardPrograms: { [key: number]: RewardProgram[] } = {
   4: [
     {
       name: 'Aludel',
-      shortName: 'ALDL',
+      shortName: 'WETH',
```
Comment (nuurcodes) [inline src/store/rewardPrograms.ts:47]: Is this short name indicative of which rewards the user gets?

File: src/hooks/useSubscriptions.ts [hunk 5]
```diff
@@ -100,16 +122,41 @@ export function useSubscriptions(
...
+              finalAmount, // value is in bonus token
+              // bonus token reward value in USD
+              15,
+            ] as BonusToken;
           })
         );
 
         const bonusTokenRewards = await bonusTokenRewardsPromise;
 
+        const rewardTokenBalanceInDai = await getTokenBalanceInDai(
```
Comment (nuurcodes) [inline src/hooks/useSubscriptions.ts:141]: I like that this is a separate hook 💯

File: src/components/modals/tx/txConfirmedFlashbotsModal.tsx [hunk 6]
```diff
@@ -10,15 +10,17 @@ import {
...
   message?: string;
 };
 
-const TxConfirmedFlashbotsModal: FC<Props> = ({ onClose, message }) => {
+const TxConfirmedFlashbotsModal: FC<Props> = ({ message }) => {
+  const { closeModal } = useModal();
+
   return (
-    <Modal isOpen={true} onClose={onClose}>
+    <Modal isOpen={true} onClose={() => closeModal()}>
```
Comment (nuurcodes) [inline src/components/modals/tx/txConfirmedFlashbotsModal.tsx:23]:
```suggestion
    <Modal isOpen={true} onClose={closeModal}>
```

File: src/components/modals/tx/txConfirmedFlashbotsModal.tsx [hunk 7]
```diff
@@ -30,7 +32,7 @@ const TxConfirmedFlashbotsModal: FC<Props> = ({ onClose, message }) => {
           <Text color='gray.200'>{message}</Text>
         </ModalBody>
         <ModalFooter>
-          <Button onClick={onClose} isFullWidth>
+          <Button onClick={() => closeModal()} isFullWidth>
```
Comment (nuurcodes) [inline src/components/modals/tx/txConfirmedFlashbotsModal.tsx:35]:
```suggestion
          <Button onClick={() => closeModal} isFullWidth>
```

File: src/components/modals/transferModal.tsx [hunk 8]
```diff
@@ -77,7 +77,11 @@ const TransferModal: FC<Props> = ({ onClose, id }) => {
             </FormControl>
           </ModalBody>
           <ModalFooter>
-            <Button isFullWidth onClick={handleTransferCrucible}>
+            <Button
+              variant='cyanbutton'
```
Comment (nuurcodes) [inline src/components/modals/transferModal.tsx:81]: Nicee, I like this variant approach

Maybe we can have a button set in the near future with 'primary, secondary, tertiary' button variants

File: src/styles/global-styles.ts (hunk 2 above)
Comment (joehawary) [reply src/styles/global-styles.ts:30]: Yes, for Internet Explorer

File: src/store/rewardPrograms.ts (hunk 4 above)
Comment (joehawary) [reply src/store/rewardPrograms.ts:47]: Should be yes :)

File: src/components/modals/transferModal.tsx (hunk 8 above)
Comment (joehawary) [reply src/components/modals/transferModal.tsx:81]: Sounds like a plan :)

File: src/components/crucible/detail/assets-detail/Subscription.tsx [hunk 9]
```diff
@@ -32,6 +45,71 @@ type Props = {
...
+        w='100%'
+        h='5px'
+        borderRadius='0 0 8px 8px'
+      />
+      <Box
+        position='absolute'
+        bg='purple.300'
+        w={completionPercent}
+        h='5px'
+        borderRadius={
```
Comment (nuurcodes) [inline src/components/crucible/detail/assets-detail/Subscription.tsx:65]: Overflow hidden on the container could be a better solution

**Review by nuurcodes — APPROVED** [review]
Looks good

**Review by nuurcodes — APPROVED** [review]

**Pull request merged**